packages/web-runtime/src/container/bootstrap.ts

```diff
@@ -30,6 +30,39 @@ export const requestConfiguration = async (path: string): Promise<RuntimeConfigu
   })
 }
 
+/**
+ * Prepare store with basic that are needed before any application gets announced.
+ *
```

Comment:
needs update after renaming it back to "announce...", something like:
.. announce and prepare vue store ...

Reply:
👍